src/main/docker/keycloak.yml

```diff
@@ -2,7 +2,7 @@
 version: '3.8'
 services:
   keycloak:
-    image: jboss/keycloak:12.0.4
+    image: jboss/keycloak:15.0.2
```

Comment:
I think we should also update realm configurations [over here](https://github.com/jhipster/jhipster-registry/tree/main/src/main/docker/realm-config)

Reply:
I have updated jhipster-realm.json with the new version.